packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -36,6 +34,10 @@ export const {
 export const enableProfilerTimer = __PROFILE__;
 export const enableProfilerCommitHooks = __PROFILE__;
 
+// TODO: These features do not yet exist in the new reconciler fork.
+export const enableDebugTracing = !__VARIANT__;
```

Comment:
What's the reason for this change? Is `enableDebugTracing` no longer a dynamic flag in www?

Reply:
I guess it's okay for it to be, since it's wrapped in `__DEV__ `checks. I'll move it back only.

Reply:
I think it's fine either way because it only triggers if the mode is set. But if it's meant to be enabled, this can just be `true`.

Reply:
It's not meant to be enabled always, because it's pretty noisy in the console. It's meant to be something we flip on manually when we're debugging.

PR #19375 is what makes the flag changes though; I've added some additional commenting about when/why you'd want to use the flag.